src/uShip.Logging.Tests/LoggingEventPropertiesBuilderTests.cs

```diff
@@ -53,6 +54,20 @@ public void Should_sanitize_creditCard()
         }
 
         [Test]
+        public void Should_sanitiz_credit_cards_from_loggable_exception()
+        {
+            string expectedValue =
+                "{\"user\":{\"************\":\"****\"},\"userPreferences\":{\"currency\":\"USD\",\"siteId\":\"UnitedStates\",\"language\":\"en-US\",\"timeZone\":\"UTC\"},\"profile\":{}}";
```

Comment:
The short answer is because of the following code which I will make a tech debt followup ticket to rewrite this

        private static readonly RegexReplacement[] SensitiveInfoPatterns =
            uShipLogging.Config.JsonReplacements
                .Cast<uShipLoggingConfigurationSection.JsonReplacementsElementCollection.AddElement>().Select(x => new JsonReplacement(x.Field))
                .Concat<RegexReplacement>(
                    uShipLogging.Config.UrlFormEncodedReplacements
                        .Cast<uShipLoggingConfigurationSection.UrlFormEncodedReplacementsElementCollection.AddElement>()
                        .Select(x => new UrlFormEncodedReplacement(x.Field)))
                .Concat(
                    uShipLogging.Config.RegexReplacements
                        .Cast<uShipLoggingConfigurationSection.RegexReplacementsElementCollection.AddElement>()
                        .Select(x => new RegexReplacement(x.Field, "************"))).ToArray();